wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxDataSource.swift

```diff
@@ -152,8 +152,11 @@ class MessageToolboxDataSource {
 
     /// Creates a label that display the status of the message.
     private func makeDetailsString() -> (NSAttributedString?, NSAttributedString?, NSAttributedString?) {
-        let deliveryStateString: NSAttributedString? = selfStatus(for: message)
         let countdownStatus = makeEphemeralCountdown()
+        let selfMessagedeliveryStateString: NSAttributedString? = selfStatus(for: message)
+        let messageDeliveryStateString: NSAttributedString? = otherUserMessageStatus(for: message)
+
+        let deliveryStateString = messageDeliveryStateString ?? selfMessagedeliveryStateString
```

Comment:
What is the reason for this change?

Reply:
We already discussed that. I will make sure Wolfang update the design specs